index.bs

```diff
@@ -838,6 +838,64 @@ interface ReceiveStream {
 ReceiveStream includes IncomingStream;
 </pre>
 
+# Protocol Mappings # {#protocol-mapping}
+
+This section specifies the [[QUIC-TRANSPORT]] protocol behavior expected from methods defined
+in this specification as well as in [[WHATWG-STREAMS]].
+
+  <table class="data">
+    <colgroup class="header"></colgroup>
+    <colgroup span=2></colgroup>
+    <thead>
+      <tr>
+        <th>Method</th>
+        <th>QUIC Protocol Action</th>
+      </tr>
+    </thead>
+    <tbody>
+      <tr>
+        <td>{{IncomingStream/abortReading}}</td>
+        <td>send STOP_SENDING with code</td>
+      </tr>
+      <tr>
+        <td>{{OutgoingStream/abortWriting}}</td>
+        <td>send RESET_STREAM with code</td>
+      </tr>
+      <tr>
+        <td>{{OutgoingStream/writable}}.abort()</td>
+        <td>send RESET_STREAM</td>
+      </tr>
+      <tr>
+        <td>{{OutgoingStream/writable}}.close()</td>
+        <td>send STREAM_FINAL</td>
+      </tr>
+      <tr>
+        <td>{{OutgoingStream/writable}}.getWriter().write()</td>
+        <td>send STREAM</td>
+      </tr>
+      <tr>
+        <td>{{OutgoingStream/writable}}.getWriter().close()</td>
+        <td>send STREAM_FINAL</td>
+      </tr>
+      <tr>
+        <td>{{OutgoingStream/writable}}.getWriter().abort()</td>
+        <td>send RESET_STREAM</td>
+      </tr>
+      <tr>
+        <td>{{IncomingStream/readable}}.cancel()</td>
+        <td>send STOP_SENDING</td>
+      </tr>
+      <tr>
+        <td>{{IncomingStream/readable}}.getReader().read()</td>
+        <td>receive STREAM or STREAM_FINAL</td>
+      </tr>
+      <tr>
+        <td>{{IncomingStream/readable}}.getReader().cancel</td>
```

Comment:
```suggestion
        <td>{{IncomingStream/readable}}.getReader().cancel()</td>
```

Reply:
Fixed. However, now that the IETF WEBTRANS WG is going to focus on Http3Transport, the reference to [[QUIC-TRANSPORT]] is no longer appropriate.